File: BaekJoon/Silver/10814/Python/10814.py
import sys

n = int(sys.stdin.readline())
people = []
for i in range(0, n):
    age, name = sys.stdin.readline().split()
    age = int(age)
    people.append(tuple([age, name]))

people.sort(key= lambda x:x[0])
for person in people:
    print(person[0], person[1], sep=' ')

File: BaekJoon/Silver/10814/C++/10814.cpp
// no.10814: 나이순 정렬 (S5)

#include <cstdio>
#include <algorithm>
#include <string>
using namespace std;

struct Person {
    int age;
    char name[101];
};

int main() {
    int n;
    scanf("%d", &n);
    Person arr[n];
    for(int i=0; i<n; i++) {
        scanf("%d %s", &(arr[i].age), arr[i].name);
    }
    stable_sort(arr, arr+n, [](Person a, Person b) {
        if(a.age < b.age) {
            return true;
        } else {
            return false;
        }
    });
    for(Person temp : arr) {
        printf("%d %s\n", temp.age, temp.name);
    }
    return 0;
}